internal/controllers/envoyfleet_resources.go

```diff
@@ -162,7 +162,7 @@ func (e *EnvoyFleetResources) generateDeployment(ctx context.Context) error {
 		ImagePullPolicy: corev1.PullIfNotPresent,
 		Command:         []string{"/bin/sh", "-c"},
 		Args: []string{
-			"envoy -c /etc/envoy/envoy.yaml --component-log-level filter:trace,ext_authz:trace --service-node $POD_NAME",
+			"envoy -c /etc/envoy/envoy.yaml --service-node $POD_NAME",
```

Comment:
How come the other flags were removed?

Reply:
I think this was something that @mbana  added while he was writing the feature. 
I just removed --component-log-level so we can catch all logs

Reply:
this is what we have in main branch  `https://github.com/kubeshop/kusk-gateway/blob/main/internal/controllers/envoyfleet_resources.go#L165`